app/components/certificate/loading.tsx

```diff
@@ -0,0 +1,18 @@
+import { Center, Flex, Box, Image, Heading } from '@chakra-ui/react';
+
+import image from 'img/undraw_Processing_re_tbdu.png';
```

Comment:
I'm unclear how Remix handles assets.  Are these supposed to go in `public/` or in the tree like this?

Reply:
We'll also need to include the license for this image, or we are breaking the license.

Reply:
I'm also not sure... Alex might know a bit more about it, because she has also have files on the `img` directory

Reply:
I was looking for someone to go read the docs, which are here https://remix.run/docs/en/v1/guides/migrating-react-router-app#asset-imports.

Reply:
Another question: does undraw not supply these as SVGs?  They will scale much better if so.